nipype/utils/nipype2boutiques.py

```diff
@@ -3,37 +3,49 @@
                         absolute_import)
 
 from builtins import str, open, bytes
-# This tool exports a Nipype interface in the Boutiques (https://github.com/boutiques) JSON format.
-# Boutiques tools can be imported in CBRAIN (https://github.com/aces/cbrain) among other platforms.
+# This tool exports a Nipype interface in the Boutiques
+# (https://github.com/boutiques) JSON format. Boutiques tools
+# can be imported in CBRAIN (https://github.com/aces/cbrain)
+# among other platforms.
 #
 # Limitations:
-#   * List outputs are not supported.
-#   * Default values are not extracted from the documentation of the Nipype interface.
-#   * The following input types must be ignored for the output path template creation (see option -t):
-#     ** String restrictions, i.e. String inputs that accept only a restricted set of values.
-#     ** mutually exclusive inputs.
-#   * Path-templates are wrong when output files are not created in the execution directory (e.g. when a sub-directory is created).
-#   * Optional outputs, i.e. outputs that not always produced, may not be detected.
+#   * Optional outputs, i.e. outputs that not always produced, may not be
+#     detected. They will, however, still be listed with a placeholder for
+#     the path template (either a value key or the output ID) that should
+#     be verified and corrected.
+#   * Still need to add some fields to the descriptor manually, e.g. url,
+#     descriptor-url, path-template-stripped-extensions, etc.
 
 import os
-import argparse
 import sys
-import tempfile
 import simplejson as json
+import six
```

Comment:
we don't use six as a dependency of nipype. this bit should use the relevant pieces from `future`
```suggestion
```

Reply:
I seem to have broken something in the build in removing this import. It fails with:
```
UnsatisfiableError: The following specifications were found to be in conflict:
  - matplotlib
Use "conda search <package> --info" to see the dependencies for each package.
```
Do you know where this could come from?

Reply:
@erinb90 the error is unrelated, and should be fixed if you fetch current master

Sorry for all these CI build inconveniences - it seems everything waited on your PR to break